### PR TITLE
Download generated API clients

### PR DESCRIPTION
**Pull request by blakeembrey**

### CODE REVIEW

File: dist/styles/api-console-light-theme.css [hunk 1]
```diff
@@ -1232,15 +1232,22 @@ th {
...
+.raml-console-meta-button-container {
+  position: relative;
+  float: right;
+}
+
+.raml-console-meta-button {
+  display: block;
   padding: 5px 8px;
-  border: solid rgba(0, 0, 0, 0.1);
+  border: 1px solid rgba(0, 0, 0, 0.1);
```
Comment (dmartinezg) [inline dist/styles/api-console-light-theme.css]: Why setting the width and overwrite it below? You are doing the same in the other theme.

Comment (blakeembrey) [inline dist/styles/api-console-light-theme.css]: Fixed.

**Comment by xaka** [comment]
Could you please demo the new feature as a GIF attachment?

**Comment by blakeembrey** [comment]
raml-client-generator-api-console

**Comment by dmartinezg** [comment]
How does this integrate with designer?
  - How does this look in designer? 
  - Are the buttons that large there too?
  - In designer shouldn't those buttons belong to the menu?

Isn't the spacing between the API title and the button a little too small?

**Comment by blakeembrey** [comment]
1. It would show like it is now in the designer
2. Yes, I think so. I just reused existing buttons but updating the design should really be another issue
3. I considered that too, but it was decided that the console was the correct space for it
4. Yes, see 2.

**Comment by xaka** [comment]
Let's merge :+1:

**Pull request merged**